app/controllers/service_provider_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ServiceProviderController < ApplicationController
+  protect_from_forgery with: :null_session
+
+  def update
+    if FeatureManagement.use_dashboard_service_providers?
+      ServiceProviderUpdater.new.run
+      SecureHeadersWhitelister.new.run
```

Comment:
I think I misunderstood how this works. Is the purpose of this endpoint to allow the dashboard app to ping the IdP when the SP config changes so that the IdP can pick it up dynamically without having to restart the server?